priv/posts/eef/20210902233320_agm-2021.md

```diff
@@ -0,0 +1,67 @@
+  "category": "eef",
+  "tags": ["eef", "agm"],
+  "datetime": "2021-09-02T23:33:20.702576Z"
+}
+---
+ EEF 2nd. Annual General Meeting
+---
+
+## Zoom Conference Call Times
+
+Thursday Sept 9th, 20:00 CET / 18:00 UTC / 09:00 USA Pacific Time
+
+**Meeting A:** EEF Annual General Meeting 2021 EMEA / Americas
+Time: Thursday, Sep 9th, 2021 20:00 CET / 18:00 UTC / 09:00 USA Pacific Time 
+
+Members will receive an email with a link and credentials for the meetings. 
+
+**Meeting B:** EEF Annual General Meeting 2021 EMEA / APAC
+Friday: Sep 10th, 2021 15:00 Japanese Time; 08:00 CET; 06:00 UTC
+
+Members will receive an email with a link and credentials for the meetings. 
+
+## Quorum Requirements
+Our Bylaws Section 3.9 require a quorum of 1/3 (one third) of the eligible voting members to be present for a meeting to be deemed quorate.
+Since:
+1. Covid-19 restrictions prohibit an in-person AGM, and
+2. Global distribution of the members make real-time synchronization due to time-zone awkward,
+
+the board decided per Section 3.1 to hold the AGM electronically. In order to comply with the three subsections of Section 3.1 that require authentication of the voting members and allowing members reasonable access to the content, we will be holding the actual voting via the Wild Apricot Members portal (authentication) and two Zoom Conferences (content access) as a mechanism of interactive feedback and Q&A.
+
+**Note:** since the AGM is electronic and virtual, Section 14.8 Electronic Voting rules apply.
+Proposed AGM Process
+1. Electronic voting will open on September 11th.
+2. No specific provision will be made for proxy voting, since voting is electronic and will take place over several days.
+3. The two Zoom calls will go through the AGM content.
+4. The content of the AGM will also be posted online.
+5. Voting members are not required to attend either of the Zoom calls but are encouraged to review the meeting content in some way before voting.
+6. All voting members of record as of September 10th are entitled and encouraged to vote.
+7. The AGM voting will be open for the 14 days period after the AGM date of September 11th.
+
+## General Purpose of any AGM
+1. Elect or affirm the board of directors.
+2. Receive information about the past year's activities, progress, achievements and milestones.
+3. Find out about the organization's financial health.
+4. Make changes to the corporation's objectives or bylaws.
+5. Receive information on future directions
+6. Ask questions
+
+## Actual Agenda for the 2nd. EEF AGM
+1. Call to order and declaration of process
+2. Approval of agenda
+3. Welcome and introductions
+4. Annual report  presentation and adoption
+* Chair's report
+* Committee reports
+5. Treasurer's  presentation and adoption
+* Report on Formal 990-EZ Filing
+* Income and Expense to date
+6. Special Resolutions  presentation, motion and voting
+* Ratification for Cohort A made in March 2021
+7. New Business / General Questions
```

Comment:
Might make sense to mention this part higher up in the announcement, this is an open floor for people who have questions about the foundation as well.

Reply:
Do you mean the entire agenda?

Reply:
Maybe adding something on line 12 like "The EEF Annual General Meeting (AGM) is an opportunity to get an update about how the Foundation is run, how it's doing, and to ask questions you might have to the board" -- Just a lede that tells people what the AGM is about without necessarily getting into the details of the agenda.

Reply:
I like it.